pipelines/README.md

```diff
@@ -1,163 +1,129 @@
 
-    [![Weather Summarizer with PaLM API](llm/palm2_weather_summarizer.jpg)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_weather_summarizer.json)
+  * G1: The description should **explicitly** explain **what** the pipeline does.
+  * G2: The description should NOT explain **how** the pipeline does it.
 
-*   [LaTeX Manuscript Helper](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_manuscript_helper.json)
-    with [PaLM API](https://developers.generativeai.google)
+    For instance, for the [weather summarizer](https://visualblocks.withgoogle.com/#/edit/_project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_weather_summarizer.json) pipeline:
 
-    [![LaTeX Manuscript Helper](llm/palm2_manuscript_helper.jpg)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_manuscript_helper.json)
+    &#9745; "Summarize the weather in a specified city into in one sentence." - a good example
 
-*   [Werewolf Game](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_werewolf.json)
-    with [PaLM API](https://developers.generativeai.google)
+    &#9746; "Weather summarizer" - a description that is too general and ambiguous [violating P1]
 
-    [![Werewolf Game](llm/palm2_werewolf.jpg)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Fpalm2_werewolf.json)
+    &#9746; "Get the weather in specified city first, and then use PaLM to summarize the generated content into one sentence." - a description that includes too many unnecessary technical details [violating P2]
 
-*   [Review Generation and Rating](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Freview_generation_and_rating.json)
-    with GPT3.5
+2. **tags** (*required*) - An array of strings containing one or more of the following possible tags:
+     * text
+     * vision
+     * other
 
-    [![Review Generation and Rating](llm/review_generation_and_rating.jpg)](https://visualblocks.withgoogle.com/#/edit/_?project_json=https:%2F%2Fraw.githubusercontent.com%2Fgoogle%2Fvisualblocks%2Fmain%2Fpipelines%2Fllm%2Freview_generation_and_rating.json)
+   For example, if your pipeline was solely around vision, you would just enter `[“vision"]` for this field.
+   If your pipeline covered multiple areas, for example a speech to text pipeline's categories might be `["audio”, “text"]` as the string array for this field.
```

Comment:
Since we don't have "audio" in the tag list, maybe we could update this example to use "vision" and "text"?